src/main/java/org/apache/maven/model/jdom/util/JDomUtils.java

```diff
@@ -255,6 +259,95 @@ public static void removeChildElement(Element parent, Element removeChild) {
     removeChild.detach();
   }
 
+  /**
+   * Remove a child of type {@link Content} and its attached comments from the parent.
+   *
+   * @param parent      the parent element.
+   * @param removeChild the child content to be removed.
+   */
+  public static void removeChildAndItsCommentFromContent(Element parent, Content removeChild) {
+    int index = parent.indexOf(removeChild);
+    if (index >= 0) {
+      LOG.debug("");
+      LOG.debug("index [{}] => REMOVE: {}", index, JDomContentHelper.contentAsString(parent.getContent(index)));
+      parent.removeContent(index);
+      index--;
+      // remove new line
+      Content elementToCheck = index >= 0 ? parent.getContent(index) : null;
+      if (index >= 0 && JDomContentHelper.isNewline(elementToCheck)) {
+        LOG.debug("NL    [{}] => REMOVE: {}", index, JDomContentHelper.contentAsString(elementToCheck));
+        simpleRemoveAtIndex(index, parent, false);
+
+        int prevIndex = index - 1;
+        if (prevIndex >= 0) {
+          // remove comment
+          LOG.debug("\t@[{}] => {}", prevIndex, JDomContentHelper.contentAsString(parent.getContent(prevIndex)));
+          int newIndex = removeCommentAtIndex(prevIndex, parent);
+          if (newIndex >= 0) {
+            LOG.debug("\t*[{}] => {}", newIndex, JDomContentHelper.contentAsString(parent.getContent(newIndex)));
+          }
+          while (newIndex >= 0 && JDomContentHelper.isComment(parent.getContent(newIndex))) {
+            LOG.debug("\t*[{}] => ALSO REMOVE: {}", newIndex, JDomContentHelper.contentAsString(parent.getContent(newIndex)));
+            newIndex = removeCommentAtIndex(newIndex, parent);
+            if (newIndex >= 0) {
+              LOG.debug("\t [{}] => NEXT       : {}", newIndex, JDomContentHelper.contentAsString(parent.getContent(newIndex)));
+            }
+            if (newIndex >= 0 && JDomContentHelper.isMultiNewLine(parent.getContent(newIndex - 1))) {
+              break;
+            }
+          }
+        }
+      } else if (index >= 0 && JDomContentHelper.isMultiNewLine(elementToCheck)) {
+        LOG.debug("ML    [{}] => *******REPLACE one newline and indentation: {}", index, JDomContentHelper.contentAsString(elementToCheck));
```

Comment:
`*******`?

Reply:
Removed in https://github.com/CoreMedia/maven-jdom-parser/pull/12/commits/1e5730206c30c7665f11ddc768d9d1e5df0eca00